qiskit_ibm/runtime/ibm_runtime_service.py

```diff
@@ -107,17 +107,24 @@ def __init__(self, provider: 'ibm_provider.IBMProvider') -> None:
         self._ws_url = provider.credentials.runtime_url.replace('https', 'wss')
         self._programs = {}  # type: Dict
 
-    def pprint_programs(self, refresh: bool = False) -> None:
+    def pprint_programs(self, refresh: bool = False, include_details: bool = False) -> None:
```

Comment:
nit - can we use `detailed` as the parameter name to match the one for printing composite job report?